pkg/nutanix/v3/v3_service.go

```diff
@@ -1818,6 +1819,24 @@ func (op Operations) ListAllUser(filter string) (*UserListResponse, error) {
 	return resp, nil
 }
 
+/*GetCurrentLoggedInUser This operation gets the user info for the currently logged in User.
+ *
+ * @param context
+ * @return *User
+ */
+func (op Operations) GetCurrentLoggedInUser(ctx context.Context) (*UserIntentResponse, error) {
+
+	path := "/users/me"
+	User := new(UserIntentResponse)
```

Comment:
Consider moving this to line 1836 so it's defined just before it actually needs to be used.